### PR TITLE
minor tweaks

### PR DESCRIPTION
**Pull request by kares**
... commits do say it all

### CODE REVIEW

**Comment by dsander** [comment]
Thanks @kares! Looks good to me.

File: app/models/agents/website_agent.rb [hunk 1]
```diff
@@ -443,7 +443,7 @@ def extraction_type(interpolated = interpolated())
     end
 
     def use_namespaces?
-      if value = interpolated.key?('use_namespaces')
```
Comment (cantino) [inline app/models/agents/website_agent.rb]: What is this change for?

Comment (cantino) [inline app/models/agents/website_agent.rb]: Oh, never mind.  I agree with this :)

File: config/initializers/delayed_job.rb [hunk 2]
```diff
@@ -16,12 +16,10 @@ class Delayed::Job
   scope :failed, -> { where("failed_at IS NOT NULL") }
 end
 
-def database_deadlocks_when_using_optimized_strategy?
+database_deadlocks_when_using_optimized_strategy = lambda do
```
Comment (cantino) [inline config/initializers/delayed_job.rb]: Is this any less polluting?  Does this make a global variable?

Comment (kares) [inline config/initializers/delayed_job.rb]: nope, its a local variable which lives in the scope of the file (does not leak elsewhere).

Comment (cantino) [inline config/initializers/delayed_job.rb]: Makes sense, thanks!

**Comment by cantino** [comment]
Thanks @kares!

**Pull request merged**